### ISSUE TITLE
Ch11: react-router-bootstrap installation fails

### ISSUE THREAD

**Issue by elstoc** [vasansr/pro-mern-stack-2#46]
Command `npm install react-router-bootstrap@0` produced an `unable to resolve dependency tree` error and should probably be replaced with `npm install react-router-bootstrap@rr-v4` in the _Navigation Bar_ section of Ch11.

At least this seemed to work for me.

**Comment by Hitendra27** [comment]
yes, I too had the same problem. Your solution worked for me as well.
Thanks a lot, elstoc.
